moveit_core/trajectory_processing/CMakeLists.txt

```diff
@@ -20,6 +23,7 @@ ament_target_dependencies(${MOVEIT_LIB_NAME}
 target_link_libraries(${MOVEIT_LIB_NAME}
   moveit_robot_state
   moveit_robot_trajectory
+  ruckig::ruckig
```

Comment:
nit, doesn't `ament_target_dependencies(... ruckig)` work? `target_link_libraries()` doesn't take care about include directories

Reply:
I have tested that. Indeed, it does not work. I think it's because ruckig is installed like a system dependency, not like a normal ROS2 pkg.